### ISSUE TITLE
Deprecation signatures placed in an ugly way

### ISSUE THREAD

**Issue by mrkkrp** [tweag/ormolu#262]
This is best reproduced by trying to ormolize `attoparsec`.

```haskell
module Data.Attoparsec
    {-# DEPRECATED "This module will be removed in the next major release." #-}
    (
      module Data.Attoparsec.ByteString
    ) where
```

becomes

```haskell
module Data.Attoparsec {-# DEPRECATED "This module will be removed in the next major release." #-}
  ( module Data.Attoparsec.ByteString
    )
where
```

Which is not very nice.

**Comment by mrkkrp** [comment]
Those annotations are also grouped incorrectly:

```haskell
-- | /Obsolete/. A type-specialized version of '*>' for 'Text'. Use
-- '*>' instead.
(.*>) :: Text -> Parser a -> Parser a
s .*> f = I.string s *> f

{-# DEPRECATED (.*>) "This is no longer necessary, and will be removed. Use '*>' instead." #-}
-- | /Obsolete/. A type-specialized version of '<*' for 'Text'. Use
-- '*>' instead.
(<*.) :: Parser a -> Text -> Parser a
f <*. s = f <* I.string s

{-# DEPRECATED (<*.) "This is no longer necessary, and will be removed. Use '<*' instead." #-}
````

**Comment by utdemir** [comment]
It looks like the second issue was fixed by #263 . I'm working on the first one.
